### PR TITLE
feat: custom stickiness on flexible rollout strategy

### PR DESCRIPTION
**Pull request by sbelzile-nexapp**
Related to https://github.com/Unleash/unleash-client-node/pull/201

### CODE REVIEW

File: specifications/10-flexible-rollout-strategy.json [hunk 1]
```diff
@@ -155,13 +171,21 @@
...
             "toggleName": "Feature.flexibleRollout.0",
             "expectedResult": false
+        },
+        {
+            "description": "Feature.flexibleRollout.customStickiness.42 should be enabled for organization=147",
+            "context": {
+                "organization": "147"
+            },
+            "toggleName": "Feature.flexibleRollout.0",
+            "expectedResult": false
```
Comment (ivarconr) [inline specifications/10-flexible-rollout-strategy.json:188]: Description says "...should be enabled" while expected result is `false`?

Comment (sbelzile-nexapp) [reply specifications/10-flexible-rollout-strategy.json]: oups... fixed

**Comment by ivarconr** [comment]
I wanted to add test cases for a few more edge cases so ended up with a separate branch: #11
